chitterModel.js

```diff
@@ -0,0 +1,33 @@
+class ChitterModel {
+  constructor() {
+    this.chits = []
+    this.session = []
+    console.log('session contents', this.session)
+  }
+
+  getChits() {
+   return this.chits 
+  }
+  
+  addChit(chit) {
+    this.chits.push(chit) 
+  }
+
+  reset() {
+    this.chits = []
+  }
+
+  setChits(chits) {
+    this.reset();
+    chits.forEach((chit) => this.chits.push(`${chit.user.handle}: ${chit.body}`));
+  }
```

Comment:
It might be easier to store everything that comes from the server into the chits array (including body, user, time, likes,...) and then, when displaying what you want to display, select what you want. In that case, this method could have just been something like this.chits = chits